api/models/index.js

```diff
@@ -0,0 +1,6 @@
+const dataSource = require('./dataSource');
+const userDao = require('./userDao');
+
+module.exports = {
+    userDao
+}
```

Comment:
```suggestion
const dataSource = require('./dataSource');
const userDao = require('./userDao');

module.exports = {
    dataSource,
    userDao
}
```
index를 활용하여 모듈들의 export, import를 편리하게, 일괄적으로 관리할 것 이라면, dataSource도 export 해주시고 app.js에서도 index를통해 require 할 수 있도록 수정해주세요~